process/sync/shardblock.go

```diff
@@ -172,6 +174,10 @@ func (boot *ShardBootstrap) applyNotarizedBlock(nonce uint64, notarizedHdrNonceH
 		return err
 	}
 
+	if header.GetRound() > boot.boostrapRoundIndex {
```

Comment:
is this called only in case of bootstrap ?

Reply:
Yes, this mechanism is called from the StartSync() method which is called only one time when the node starts and only is it has something in the storage